doc/releases/changelog-dev.md

```diff
@@ -116,54 +113,53 @@
 
-  A method has been added that takes a directed multigraph with `MeasureNode` and
-  `PrepareNode` placeholders and fragments into subgraphs and a communication graph.
-  [(#2153)](https://github.com/PennyLaneAI/pennylane/pull/2153)
+  - A method has been added that takes a directed multigraph with `MeasureNode` and
+    `PrepareNode` placeholders and fragments into subgraphs and a communication graph.
+    [(#2153)](https://github.com/PennyLaneAI/pennylane/pull/2153)
 
-  A method has been added that takes a directed multigraph with `MeasureNode`
-  and `PrepareNode` placeholder nodes and converts it into a tape.
-  [(#2165)](https://github.com/PennyLaneAI/pennylane/pull/2165)
+  - A method has been added that takes a directed multigraph with `MeasureNode`
+    and `PrepareNode` placeholder nodes and converts it into a tape.
+    [(#2165)](https://github.com/PennyLaneAI/pennylane/pull/2165)
 
-  A differentiable tensor contraction function `contract_tensors` has been
-  added.
-  [(#2158)](https://github.com/PennyLaneAI/pennylane/pull/2158)
+  - A differentiable tensor contraction function `contract_tensors` has been
+    added.
+    [(#2158)](https://github.com/PennyLaneAI/pennylane/pull/2158)
   
-  A method has been added that expands a quantum tape over `MeasureNode` and `PrepareNode`
-  configurations.
-  [(#2169)](https://github.com/PennyLaneAI/pennylane/pull/2169)
+  - A method has been added that expands a quantum tape over `MeasureNode` and `PrepareNode`
+    configurations.
+    [(#2169)](https://github.com/PennyLaneAI/pennylane/pull/2169)
 
-  The postprocessing function for the `cut_circuit` transform has been added.
-  [(#2192)](https://github.com/PennyLaneAI/pennylane/pull/2192)
+  - The postprocessing function for the `cut_circuit` transform has been added.
+    [(#2192)](https://github.com/PennyLaneAI/pennylane/pull/2192)
 
-  The `cut_circuit` transform has been added.
-  [(#2216)](https://github.com/PennyLaneAI/pennylane/pull/2216)
+  - The `cut_circuit` transform has been added.
+    [(#2216)](https://github.com/PennyLaneAI/pennylane/pull/2216)
 
-  A class `CutStrategy` which acts as an interface and coordinates device/user
-  constraints with circuit execution requirements to come up with the best sets
-  of graph partitioning parameters.
-  [(#2168)](https://github.com/PennyLaneAI/pennylane/pull/2168)
+  - A class `CutStrategy` which acts as an interface and coordinates device/user
+    constraints with circuit execution requirements to come up with the best sets
+    of graph partitioning parameters.
+    [(#2168)](https://github.com/PennyLaneAI/pennylane/pull/2168)
 
-  A suite of integration tests has been added.
-  [(#2231)](https://github.com/PennyLaneAI/pennylane/pull/2231)
-  [(#2234)](https://github.com/PennyLaneAI/pennylane/pull/2234)
-  [(#2251)](https://github.com/PennyLaneAI/pennylane/pull/2251)
+  - A suite of integration tests has been added.
```

Comment:
Out of scope here, but wondering: would we want to have this in the changelog?

Reply:
I'm guessing that @trbromley and @anthayes92 may rewrite this changelog entry prior to release, to position more as 'new feature' rather than a list of PR descriptions?